tcp_check/datadog_checks/tcp_check/tcp_check.py

```diff
@@ -103,10 +105,10 @@ def check(self, instance):
             self.log.debug("%s:%d is UP", self.addr, self.port)
             self.report_as_service_check(AgentCheck.OK, 'UP')
 
-        if self.response_time:
+        if self.response_time and response_time is not None:
```

Comment:
I find confusing that both vars are called `response_time` can we rename the object member to `collect_response_time`?

Reply:
Fixed.